### DISCUSSION TITLE
UX: differenciate username from content

### DISCUSSION THREAD

**Discussion by Geobert** [cheeaun/phanpy#566] [Ideas]
I find it hard to differenciate the username from the post’s content at they are in the same font. What about putting the username in bold, like the original front?

EDIT: just saw that we have `font-weight` to 500 but I can’t see the diff. I start to see it at 600

**Comment by Fastidious** [comment]
The user name is in bold. Am I missunderstanding?

![image](https://github.com/cheeaun/phanpy/assets/8352292/b22275a2-1d91-403c-955b-468c009a9262)

**Reply by Geobert** [reply]
I don’t have this but I’m in dark mode

![Screenshot 2024-06-06 at 20-52-15 Phanpy](https://github.com/cheeaun/phanpy/assets/72570/ff4ac42b-9ad3-42a2-8443-5a31f98e669e)

Firefox stable 126.0.1 on Windows 11

**Reply by Fastidious** [reply]
This is what I see on dark mode (which I used on mobile, on auto mode):

![image](https://github.com/cheeaun/phanpy/assets/8352292/52c0e65a-efff-4df4-b8e9-a740879ecade)

Chrome version 125.0.6422.112 (Official Build) (64-bit)

**Reply by Fastidious** [reply]
Sadly, I don't have Windows to test. It might be related to the OS.

**Reply by Geobert** [reply]
Yeah I tried light to see and can’t see the 500 font-weight, I applied a script to set 600 to get this:

![Screenshot 2024-06-06 at 20-58-28 Phanpy](https://github.com/cheeaun/phanpy/assets/72570/d4d171d8-8453-4001-bbbd-c40ebf1f6de8)
